roxie/ccd/ccdprotocol.cpp

```diff
@@ -1837,26 +1837,39 @@ class RoxieSocketWorker : public ProtocolQueryWorker
                 if (httpHelper.isHttp())
                     httpHelper.setUseEnvelope(extractor.isSoap);
             }
+
+            if (!queryName)
+            {
+                // or should we set("") ?  Is an empty queryName ever valid ?
+                if (doTrace(traceSockets, TraceFlags::Max))
+                {
+                    StringBuffer b;
```

Comment:
Not needed - did you intend to get the peer name or similar?

Reply:
Peer is often an unhelpful vip/proxy/load-balancer address, but yes I should print the peer, and perhaps also the first 100 chars of rawText.str() ?  Or check if its an http request and then print headers etc.
I was thinking of a follow-on PR for more verbose tracing of this type of error because Max isnt something likely to be set on in production.

Reply:
removed StringBuffer b line